yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1001,38 +1000,54 @@ func fetchColumnsWithUnsupportedDataTypes() ([]utils.TableColumnsDataTypes, []ut
 		err := rows.Scan(&columnDataTypes.SchemaName, &columnDataTypes.TableName,
 			&columnDataTypes.ColumnName, &columnDataTypes.DataType)
 		if err != nil {
-			return nil, nil, fmt.Errorf("error scanning rows: %w", err)
+			return nil, nil, nil, fmt.Errorf("error scanning rows: %w", err)
 		}
 
 		allColumnsDataTypes = append(allColumnsDataTypes, columnDataTypes)
 	}
 
-	var sourceUnsupportedDataTypes []string
-	var liveMigrationUnsupportedDataTypes []string
+	var sourceUnsupportedDataTypes, liveMigrationUnsupportedDataTypes, liveMigrationWithFForFBUnsupportedDatatypes []string
+
 	switch source.DBType {
 	case POSTGRESQL:
 		sourceUnsupportedDataTypes = srcdb.PostgresUnsupportedDataTypes
 		liveMigrationUnsupportedDataTypes, _ = lo.Difference(srcdb.PostgresUnsupportedDataTypesForDbzm, srcdb.PostgresUnsupportedDataTypes)
+		unsupportedDataTypesForDbzmYBOnly, _ := lo.Difference(srcdb.YugabyteUnsupportedDataTypesForDbzm, srcdb.PostgresUnsupportedDataTypes)
+		liveMigrationWithFForFBUnsupportedDatatypes, _ = lo.Difference(unsupportedDataTypesForDbzmYBOnly, liveMigrationUnsupportedDataTypes)
 	case ORACLE:
 		sourceUnsupportedDataTypes = srcdb.OracleUnsupportedDataTypes
 	default:
 		panic(fmt.Sprintf("invalid source db type %q", source.DBType))
 	}
-
 	// filter columns with unsupported data types using sourceUnsupportedDataTypes
 	for i := 0; i < len(allColumnsDataTypes); i++ {
 		//Using this ContainsAnyStringFromSlice as the catalog we use for fetching datatypes uses the data_type only
 		// which just contains the base type for example VARCHARs it won't include any length, precision or scale information
 		//of these types there are other columns available for these information so we just do string match of types with our list
-		if utils.ContainsAnyStringFromSlice(sourceUnsupportedDataTypes, allColumnsDataTypes[i].DataType) {
+		splits := strings.Split(allColumnsDataTypes[i].DataType, ".")
+		typeName := splits[0] //using typename only for the cases we are checking it from the static list of type names
+		if len(splits) > 1 {
+			//only type without schema in case of udt and ARRAY
+			typeName = strings.TrimSuffix(splits[1], "[]")
+		}
+		if utils.ContainsAnyStringFromSlice(sourceUnsupportedDataTypes, typeName) {
 			unsupportedDataTypes = append(unsupportedDataTypes, allColumnsDataTypes[i])
 		}
-		if utils.ContainsAnyStringFromSlice(liveMigrationUnsupportedDataTypes, allColumnsDataTypes[i].DataType) {
+		if utils.ContainsAnyStringFromSlice(liveMigrationUnsupportedDataTypes, typeName) {
 			unsupportedDataTypesForLiveMigration = append(unsupportedDataTypesForLiveMigration, allColumnsDataTypes[i])
 		}
+		if utils.ContainsAnyStringFromSlice(liveMigrationWithFForFBUnsupportedDatatypes, typeName) ||
+			utils.ContainsAnyStringFromSlice(compositeTypes, allColumnsDataTypes[i].DataType) || // if type is UDT
```

Comment:
We are using compositeTypes here (that is dereived from schema?). 
On one hand, that makes sense because there will be consistency, but on the other hand, shouldn't we just rely on whatever information is present in assessmentDB?

Reply:
Also, compositeTypes will be fully qualified, is that also the case for the datatypes we will fetch from assessment DB? Not sure.

Reply:
Yes, for UDT and ARRAY type we make them qualified

Reply:
> On one hand, that makes sense because there will be consistency, but on the other hand, shouldn't we just rely on whatever information is present in assessmentDB?

I was not sure how I can get the information about the UDT / Enum from there. That's why I am using the `compositeTypes / enumTypes` from analyze